app/models/case_log.rb

```diff
@@ -51,6 +51,17 @@ class CaseLog < ApplicationRecord
                            end
                          }
 
+  scope :filter_by_id, ->(id) { where(id:) }
+  scope :filter_by_tenancy_code, ->(code) { where(tenancy_code: code) }
```

Comment:
I think we want these to be case insensitive (they are for searching user by name/email, and org by name)